nginx/conf.d/spider.conf

```diff
@@ -0,0 +1,29 @@
+map $http_user_agent $bad_client {
+    default 0;
+  ~*(360Spider)  1;
+  ~*(aiHitBot|AhrefsBot) 1;
+  ~*(betaBot|BlackWidow|Bolt|BLEXBot|BUbiNG) 1;
+  ~*(CazoodleBot|CPython|CCBot|ChinaClaw|Curious|CRAZYWEBCRAWLER|Custo) 1;
+  ~*(Default|DIIbot|DISCo|discobot) 1;
+  ~*(Exabot|eCatch|ecxi|EirGrabber|EmailCollector|EmailSiphon|EmailWolf|ExtractorPro|EyeNetIE) 1;
+  ~*(FlashGet|Findxbot) 1;
+  ~*(GetRight|GetWeb!|Go!Zilla|Go-Ahead-Got-It|Go.*package.*|GrabNet|Grafula|GT::WWW|GuzzleHttp) 1;
+  ~*(heritrix|HaosouSpider|HMView|HTTP::Lite|HTTrack) 1;
```

Comment:
> Do we really want to include that in our distro? Sorry, its a custom decision which may (and will) affect search performance and i would never recommend to do this for the website without strict reason. Can we move all this crap out?

lol, if you need search bots to to hit and index all these url `wishlist|customer|catalogsearch|newsletter|contact|sendfriend|catalog/product_compare|review/product/post|(fire|one.+)?checkout`
go ahead remove it on your server/shop configuration...